bundles/org.openhab.misc.addonsuggestionfinder/README.md

```diff
@@ -0,0 +1,70 @@
+# AddonSuggestionFinder Addon
+
+This is a special addon that implements an `AddonInfoProvider` service containing information about suggested Addons that could potentially be installed.
+It allows developers to include information in their own addons so that the system can scan the user's network to discover potential addons that can automatically be installed.
+
+## Addon Developer Notes
+
+If you want to your addon to scan the user's system then you need to include additional fields in your `src/main/resources/OH-INF/addon.xml` file.
```

Comment:
This should be documented here, i.e. through the openhab-docs repository:
https://www.openhab.org/docs/developer/addons/addon.html#xml-structure-for-add-on-definitions

A link could be provided here, but probably the contents should be removed to avoid double maintenance.

Reply:
Agreed. But let us keep it here for now, until we make more progress on the total package.